cypress/integration/11:workflow/transitions/1:create/index.test.js

```diff
@@ -57,38 +52,47 @@ MultiSteps([
     Given,
     And,
 ], 'I open {string} page', (page) => {
-    openPage({
+    cy.openPage({
         page,
-        requestName,
+        requestAlias: `@${requestName}_GET`,
     });
-    cy
-        .wait(`@${requestWorkflowName}_GET`)
-        .its('response.statusCode')
-        .should('eq', 200);
-});
-
-MultiSteps([
-    Then,
-    And,
-], 'I send a {string} request and status code should be {int}', (reqType, status) => {
-    sendRequest({
-        requestName,
-        reqType,
-        status,
+    cy.checkRequest({
+        requestAlias: `@${requestWorkflowName}_GET`,
+        statusCode: 200,
     });
 });
 
 MultiSteps([
     And,
     Then,
     When,
-], 'On {string} I can see column with {string} value', (gridId, searchValue) => {
+], 'On {string} in row {int} I create transition from {string} to {string}', (gridId, rowId, from, to) => {
+    const statuses = [];
     cy
         .get(`[data-cy=${gridId}]`)
         .should('be.visible')
         .as('grid');
+
     cy
         .get('@grid')
         .find('.workflow-designer-header-layer-cell__title span')
-        .contains(new RegExp(`${escapeStringRegexp(searchValue)}$`, 'g'));
+        .each(($el) => {
+            statuses.push($el.text().trim());
+        }).then(() => {
+            const fromIndex = statuses.findIndex(status => status === from);
+            const toIndex = statuses.findIndex(status => status === to);
+
+            cy
+                .get('@grid')
```

Comment:
Code duplication? Doing same action twice?

Reply:
save `@grid` means that this is an alias for an already saved item , but I'll make it even simpler.